dags/hindawi/xml_extractors.py

```diff
@@ -18,12 +18,15 @@ def _get_extracted_value(self, node):
             raise RequiredFieldNotFoundExtractionError(self.source)
         if node is None:
             return self.default_value
-        pattern = re.compile(r"[\n\t]*")
+        pattern = re.compile(r"[\n\t]{0,}" + r"\s{2,}")
         extracted_value = "".join(
-            [pattern.sub("", node.text) or ""]
-            + [pattern.sub("", ET.tostring(el).decode("ascii")) for el in node]
+            [node.text or ""] + [ET.tostring(el).decode("ascii") for el in node]
         )
-        return self.extra_function(extracted_value)
+        final_value = self.extra_function(pattern.sub("", extracted_value))
+        self.logger.info(
+            "Extracted value field={self.destination}, value={final_value}"
```

Comment:
```suggestion
            "Extracted value", field={self.destination}, value={final_value}"
```

Reply:
https://www.structlog.org/en/stable/why.html#easier-logging

Reply:
A, I see. I think it has to be without curly brackets